occurrences/tests/test_models.py

```diff
@@ -42,3 +42,26 @@ def test_enrolment_creation(mock_get_event_data):
     EnrolmentFactory()
     assert Occurrence.objects.count() == 1
     assert StudyGroup.objects.count() == 1
+
+
+@pytest.mark.django_db
+def test_occurrence_seat_taken():
+    EnrolmentFactory()
+    occurrence = Occurrence.objects.all()[0]
+    study_group = occurrence.study_groups.all()[0]
+    assert occurrence.seats_taken > 0
+    assert (
+        occurrence.seats_taken == study_group.group_size + study_group.amount_of_adult
+    )
+
+
+@pytest.mark.django_db
+def test_occurrence_seat_approved():
+    EnrolmentFactory(status=Enrolment.STATUS_APPROVED)
+    occurrence = Occurrence.objects.all()[0]
```

Comment:
Similar to [L50](https://github.com/City-of-Helsinki/palvelutarjotin/pull/105/files#diff-7aaed390c4a28b80908979c3cdb2208c8c353977f71781342c59f697d20b7c3dR50)